src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+            case BYE:
+                printExit();
+                isContinue = false;
+                break;
+            case LIST:
+                System.out.println("____________________________________________________________");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.println(" " + (i+1) + ". " + tasks.get(i).printStatus());
+                }
+                System.out.println("____________________________________________________________");
+                break;
+            case TODO:
+                ToDo newToDo = new ToDo(line.substring(6));
+                tasks.add(newToDo);
+                System.out.println(" (+) Added: "+ newToDo.printStatus());
+                System.out.println(" (!) You have " + tasks.size() + " tasks in the list");
+                break;
+            case DEADLINE:
+                String desc = line.substring(10, line.indexOf("-by")).strip();
+                String time = line.substring(line.indexOf("-by") + 4).strip();
+                Deadline newDeadline = new Deadline(desc, time);
+                tasks.add(newDeadline);
+                System.out.println(" (+) Added: "+ newDeadline.printStatus());
+                System.out.println(" (!) You have " + tasks.size() + " tasks in the list");
+                break;
+            case EVENT:
+                String des = line.substring(7, line.indexOf("-from")).strip();
+                String from = line.substring(line.indexOf("-from") + 6, line.indexOf("-to")).strip();
+                String to = line.substring(line.indexOf("-to") + 4).strip();
+                Event newEvent = new Event(des, from , to);
+                tasks.add(newEvent);
+                System.out.println(" (+) Added: "+ newEvent.printStatus());
+                System.out.println(" (!) You have " + tasks.size() + " tasks in the list");
+                break;
+            case DONE:
+                String[] values = line.split(" ");
+                int value = Integer.parseInt(values[1]);
+                if (value > tasks.size() || value < 1) {
+                    System.out.println(" (!) Invalid Value! ");
+                } else {
+                    tasks.get(value - 1).setStatus(true);
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" (+) Marked as Done: "+ tasks.get(value - 1).printStatus());
+                    System.out.println("____________________________________________________________");
+                }
+            default:
+                System.out.println(" (!) Invalid Value! ");
+            }
+
+//            } else if (m.find()) {
+//                String[] values = cmd.split(" ");
+//                int value = Integer.parseInt(values[1]);
+//
+
+        }
+    }
+
+    public static Command getCommand(String input) {
+        Command cmd = Command.UNKNOWN;
+        if (input.equals("/bye")) {
```

Comment:
As mentioned above, these statements can be replaced by constants. Also, switch-case can be used here